apps/meteor/server/services/federation/infrastructure/rocket-chat/adapters/File.ts

```diff
@@ -23,7 +23,7 @@ export class RocketChatFileAdapter {
 			try {
 				const { files, attachments } = await parseFileIntoMessageAttachments(uploadedFile, internalRoomId, internalUser);
 
-				resolve({ files, attachments });
+				resolve({ files: files as FileProp[], attachments });
```

Comment:
If you're casting files as FileProp, why not making `parseFileINtoMessageAttachment` return a `FileProp[]`?

Reply:
makes sense 🤙